src/styles/blocks/page.scss

```diff
@@ -0,0 +1,5 @@
+// .page {
```

Comment:
Remove this file if you did not use it. 
Don't leave comments in the production code.